src/shared/components/Gallery/Gallery.tsx

```diff
@@ -10,15 +13,35 @@ export type GalleryProps = {
   className?: string
 } & CarouselProps
 
+interface ImperativeHandleData {
+  getPrevArrowProps: () => object
+  getNextArrowProps: () => object
+}
```

Comment:
We are using `type` everywhere, so I think we should stick to it for consistency.
Also, not sure, but we could replace `object` with `React.ComponentProps<typeof Arrow>`
```tsx
type ImperativeHandleData = {
  getPrevArrowProps: () => React.ComponentProps<typeof Arrow>
  getNextArrowProps: () => React.ComponentProps<typeof Arrow>
}
```
(sorry, wanted to add a suggestion, but for some reason I don't see the button on github)

Reply:
You are ofcourse righr, it was temporary and tbh I forgot about it 😉. I think your suggestion might do a job